javascript/javascript1/week2/studentManager.js

```diff
@@ -0,0 +1,35 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  if (getNumberOfStudents(class07Students) < 6) {
+    if (studentName == " ") {
```

Comment:
what if `studentName` is `""` or with more than one space `"     "`
you can solve that using `trim()` like
```js
if (studentName.trim() === "") {
```
trim will remove all emply spaces before compare